Modules/Server/Tests/PocketCastsServerTests/SettingsStoreTests.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+@testable import PocketCastsServer
+import PocketCastsUtils
+
+class SettingsStoreTests: XCTestCase {
+
+    struct TestType: JSONCodable {
+        var name: String
+    }
+
+    private let userDefaultsSuiteName = "PocketCastsServer-SettingsStoreTests"
+
+    override func setUp() {
+        super.setUp()
+        UserDefaults.standard.removePersistentDomain(forName: userDefaultsSuiteName)
+    }
+
+    func testStore() throws {
+        let defaults = try XCTUnwrap(UserDefaults(suiteName: userDefaultsSuiteName), "User Defaults suite should load")
```

Comment:
I think this is a great idea! I'd still like to include the assertions in each test case because I want to make it easy to see exactly where something has failed instead of ending up with a failed unwrap or other error without knowing where things went wrong. In order to reduce the repeated code, I went ahead and moved the "data" elements (`defaultsKey`, `initialName`, and `changedName`) into private properties to reduce that boilerplate.